hive_flutter/lib/src/adapters/time_adapter.dart

```diff
@@ -1,17 +1,11 @@
-import 'package:flutter/material.dart';
-import 'package:hive/hive.dart';
+part of '../../adapters.dart';
 
 class TimeAdapter extends TypeAdapter<TimeOfDay> {
   @override
-  TimeOfDay read(BinaryReader reader) {
-    final time = reader.read() as TimeOfDay;
-    return time;
-  }
+  TimeOfDay read(BinaryReader reader) => reader.read() as TimeOfDay;
```

Comment:
Wait, does it really works? 😅 if yes, why does it works?

Reply:
Wait, why wouldn't it work? I'm concerned now 😅

It's written the exact same way it was before, I just turned it into an arrow function.

Should I test it in an example app? Both tests are passing tho...

Reply:
Yes, it doesn't work... I'm going to update it ASAP.

Reply:
But it wasn't working before either 🤷‍♂️

Reply:
> It's written the exact same way it was before, I just turned it into an arrow function.

So, it wasn't even working before 😅